apps/alpha-test-app/src/views/HomeView.vue

```diff
@@ -283,12 +287,29 @@ const singleSelectState = ref("Apple");
             :skeleton="useSkeleton"
           />
 
+          <OnyxTable v-if="show('OnyxTable')">
+            <thead>
+              <tr>
+                <th>Items</th>
+                <th>Expenditure</th>
```

Comment:
Table headers should be singular :) Also I think a simpler example like "Item" and "Price" is probably easier to understand than "Expenditure"

Reply:
Same for the Storybook and component test example